README.md

```diff
@@ -24,63 +26,78 @@ In order for the DSX Local to be able to utilize the resources (spark, compute)
 
 > **What is IBM DSX Local?** DSX Local is an on premises solution for data scientists and data engineers. It offers a suite of data science tools that integrate with RStudio, Spark, Jupyter, and Zeppelin notebook technologies. And yes, it can be configured to use HDP, too.
 
-> **What is the IBM DSXHI?** DSX Hadoop Integration Service (DSXHI) is a service that can be installed on a Hadoop edge node to allow DSX Local Version 1.2 or later clusters to securely access data residing on the Hadoop cluster, submit interactive Spark jobs, build models, and schedule jobs that run as a YARN application on the Hadoop cluster.
+> **What is the IBM DSXHI?** DSX Hadoop Integration Service (DSXHI) is a service that can be installed on a Hadoop edge node to allow DSX Local (version 1.2 or later) clusters to securely access data residing on the Hadoop cluster, submit interactive Spark jobs, build models, and schedule jobs that run as a YARN application on the Hadoop cluster.
 
-This repo contains 8 jupyter notebooks and 6 scripts that are used to illustarte the three scenarios mentioned above.
+This code pattern contains 8 jupyter notebooks and 6 scripts. Here is a view of the notebooks as shown by the DSX Local UI:
 
 ![](doc/source/images/Jupyter-notebooks-list.png)
 
-**Scenario 1 (Run pyspark model in remote HDP cluster)**: <br>
-- Spam Filter on local spark.ipynb (pyspark in local) <br>
-- Spam Filter on remote spark.ipynb (pyspark in HDP) <br>
+As mentioned earlier, this code pattern offers three examples of how to develope, train, and deploy a Spam Filter Model. For each example, multiple notebooks will be provided to show both a local (DSX Local) and remote (HDP cluster) solution. The following lists the associated notebooks for each:
+
+* Develop and train a Spam Filter using pyspark.
+
+  `"Spam Filter on local spark"`
+    - Load the spam collection dataset using spark context in DSX Local.
+    - Use Spark Data Pipeline to extract the TF-IDF features and use Spark MLlib to train the Spam Filter pyspark model locally.
```

Comment:
nit - I see a mix of "Spark" and "spark".  Some look intentional but there are some that I can't explain so you might want to take a pass at those.

Reply:
I'm going with uppercase